### PR TITLE
Fix packages broken by GCC 5 moving from -std=gnu89 to -std=gnu11 by default

### PR DESCRIPTION
**Pull request by SoapZaNet**
I've fixed a number of ebuilds that implicitly assumed -std=gnu89 and failed with missing symbols with GCC 5 changing the default C standard.

### CODE REVIEW

**Comment by mgorny** [comment]
@gentoo/gnome

**Comment by mgorny** [comment]
@SoapZaNet btw why did you need those gnu89 flags? clang?

**Comment by mgorny** [comment]
Also, would be good to report them as appropriate upstream. Though I think gnomemm pacakges don't need that since they should be aware of their own actions ;-).

**Comment by SoapZaNet** [comment]
@mgorny, GCC 5 switches to -std=c11 by default, so it gives the same inline errors as clang previously.

**Comment by mgorny** [comment]
Good to know.

**Comment by EvaSDK** [comment]
ack for libgnomecanvasmm, probably for gtkglextmm too. For glib-1 and gtk+-1 I will check what kind of error it raises first as I have seen no problem reported yet.

**Comment by mgorny** [comment]
@EvaSDK, it is new to gcc-5.

**Comment by mgorny** [comment]
@ZeroChaos, @robbat2, @steev, ack at -std=gnu89 for spectools?

**Comment by steev** [comment]
ack

**Comment by EvaSDK** [comment]
ack for glib:1 and gtk+:1 also I wonder why a bump would be needed.

**Comment by SoapZaNet** [comment]
No bump is needed, I just did it because @mgorny told me to :smile:

**Comment by EvaSDK** [comment]
Well, for C++ libs, there might be a good reason, I have not enough knowledge to tell right now, but for C libs I don't think this is necessary here.

**Comment by mgorny** [comment]
Yes, it's not necessary for C, esp. that we're pretty much setting the default anyway.

**Comment by jlec** [comment]
embassy-vienna is good to go.

File: x11-libs/gtk+/gtk+-1.2.10-r13.ebuild [hunk 1]
```diff
@@ -49,6 +49,8 @@ src_prepare() {
 }
 
 multilib_src_configure() {
+	append-cflags -std=gnu89
```
Comment (mgorny) [inline x11-libs/gtk+/gtk+-1.2.10-r13.ebuild]: This appends it twice or more when the func is called multiple times. You either need to define full src_configure() and call multilib-minimal_src_configure there, or you can resort to src_prepare in this case.

Comment (mgorny) [inline x11-libs/gtk+/gtk+-1.2.10-r13.ebuild]: Or `local -x CFLAGS="${CFLAGS} -std=gnu89"

**Pull request merged**